socorro/external/postgresql/report.py

```diff
@@ -84,7 +84,7 @@ def get_list(self, **kwargs):
                     '%s is not a recognized sort order key' % sort_order['key']
                 )
             sort_order['direction'] = 'ASC'
-            if 'reverse' in kwargs:
+            if str(kwargs.get('reverse', '')).lower() == 'true':
```

Comment:
That looks like a non-trivial change to the API of report/list/. Why did you make it?